examples/hoa-renderer.html

```diff
@@ -0,0 +1,171 @@
+    var hoaRenderer;
+    var audioSources = [
+      {name : 'pt1', url : 'resources/single_source_toa_pt1.wav'},
+      {name : 'pt2', url : 'resources/single_source_toa_pt2.wav'},
+      {name : 'pt3', url : 'resources/single_source_toa_pt3.wav'},
+      {name : 'pt4', url : 'resources/single_source_toa_pt4.wav'}];
+    var audioBufferPlayback = new Array(4);
+    var audioBufferPlaybackStartTime = 0;
+    var audioBufferPlaybackStopTime = 0;
+    var audioBufferPlaybackOffset = 0;
+    var buttonPlayback;
+    var audioBuffers;
+    var gainOut;
+
+    function crossProduct(a, b) {
+      c = [];
+      c[0] = a[1] * b[2] - a[2] * b[1];
+      c[1] = a[2] * b[0] - a[0] * b[2];
+      c[2] = a[0] * b[1] - a[1] * b[0];
+      return c;
+    }
+
+    function normalize(a) {
+      var n = Math.sqrt(a[0] * a[0] + a[1] * a[1] + a[2] * a[2]);
+      a[0] /= n;
+      a[1] /= n;
+      a[2] /= n;
+      return a;
+    }
+
+    function adjustGain() {
+      var gainDB = document.getElementById("gain").value;
+      document.getElementById('gainLabel').innerHTML = gainDB;
+      var gainLin = Math.pow(10, gainDB / 20);
+      gainOut.gain.value = gainLin;
+    }
+
+    function rotateScene() {
+      var azi = document.getElementById('azi').value;
+      var ele = document.getElementById('ele').value;
+      document.getElementById('aziLabel').innerHTML = azi;
+      document.getElementById('eleLabel').innerHTML = ele;
+
+      // Standard View Matrix
+      var matrix = new Array(9);
+      var beta = -azi / 180 * Math.PI;
+      var alpha = -ele / 180 * Math.PI;
+
+      matrix[0] = Math.cos(alpha) * Math.cos(beta);
+      matrix[1] = Math.sin(alpha) * Math.cos(beta);
+      matrix[2] = -Math.sin(beta);
+
+      matrix[3] = -Math.cos(alpha);
+      matrix[4] = Math.cos(alpha);
+      matrix[5] = 0;
+
+      matrix[6] = Math.cos(alpha) * Math.sin(beta);;
+      matrix[7] = Math.sin(alpha) * Math.sin(beta);
+      matrix[8] = Math.cos(beta);
+
+      hoaRenderer.setRotationMatrix(matrix);
+    }
+
+    var onLoad = function () {
+      var audioContext = new AudioContext();
+
+      var merger = audioContext.createChannelMerger(16);
+      var splitters = new Array(4);
+
+      audioBuffers = Omnitone.loadAudioBuffers(audioContext, audioSources);
+      audioBuffers.then(function (result) {
+        for (var i = 0; i < 4; i++) {
+          audioBuffers[audioSources[i].name] = result.get(audioSources[i].name);
+          splitters[i] = audioContext.createChannelSplitter(4);
+          for (var j = 0; j < 4; j++) {
+            splitters[i].connect(merger, j, i * 4 + j);
+          }
+        }
+      });
+
+      buttonPlayback = document.getElementById('eButtonPlayback');
+      buttonPlayback.onclick = function (event) {
+        if (event.target.textContent === 'Play') {
+          event.target.textContent = 'Pause';
+          audioBufferPlaybackOffset = audioBufferPlaybackOffset + audioBufferPlaybackStopTime - audioBufferPlaybackStartTime;
```

Comment:
We're following Google/Chromium JS style. The code needs to be wrapped at 80 cols.